### PR TITLE
Fix language selection

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/tabs/landing.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ landing.initialize = function (callback) {
             bottomSection.append(` <a href="#" lang="${element}" i18n="language_${element}"></a>`);
         });
 
-        bottomSection.find('a').each((index) => {
+        bottomSection.find('a').each(function(index) {
```
Comment (blckmn) [inline src/js/tabs/landing.js:41]:
```suggestion
        bottomSection.find('a').each((index, element) => {
```

File: src/js/tabs/landing.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ landing.initialize = function (callback) {
...
             let element = $(this);
```
Comment (blckmn) [inline src/js/tabs/landing.js:42]:
```suggestion
```

Comment (haslinghuis) [reply src/js/tabs/landing.js:42]: Suggestions did not work :(

File: src/js/tabs/landing.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ landing.initialize = function (callback) {
...
             element.click(() => {
```
Comment (blckmn) [inline src/js/tabs/landing.js:43]:
```suggestion
            $(element).click(() => {
```

Also need to change this (if passing the param).

**Comment by Pancronos** [comment]
This one is ok, can change language.

File: src/js/tabs/landing.js (hunk 3 above)
Comment (haslinghuis) [reply src/js/tabs/landing.js]: Updated.

**Review by nerdCopter — APPROVED** [review]
`1ebed155` works.

**Review by blckmn — APPROVED** [review]

**Pull request merged**